pkg/artifactory/resource/repository/remote/resource_artifactory_remote_helm_repository.go

```diff
@@ -19,7 +19,7 @@ var HelmRemoteSchema = func(isResource bool) map[string]*schema.Schema {
 				Type:             schema.TypeString,
 				Optional:         true,
 				Default:          "",
-				ValidateDiagFunc: validation.ToDiagFunc(validation.Any(validation.IsURLWithHTTPorHTTPS, validation.StringIsEmpty)),
+				ValidateDiagFunc: validation.ToDiagFunc(validation.Any(validation.IsURLWithScheme([]string{"http", "https", "oci"}), validation.StringIsEmpty)),
```

Comment:
We need a test for the `oci` verification. Please paste your `oci` link [here](https://github.com/jfrog/terraform-provider-artifactory/blob/204ce779b091ecf76a9a033e95d70e60dd48516d/pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go#L370C13-L370C13) and make sure test passes.